include/polo/utility/custom_termination.hpp

```diff
@@ -0,0 +1,30 @@
+#ifndef POLO_UTILITY_CUSTOMTERM_HPP_
+#define POLO_UTILITY_CUSTOMTERM_HPP_
+
+namespace polo {
+namespace utility {
+namespace terminator {
+template <class value_t, class index_t> struct custom {
+  using func_t = bool *(*)(const index_t, const value_t, const value_t *,
```

Comment:
return value should be `bool`, not `bool *`. I have fixed this, too. I will add the commit behind yours.